nodejs/spa/src/main/webapp/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Keycloak Single-Page Application Example</title>
+    <script type="text/javascript" src="KC_URL/js/keycloak.js"></script>
+    <script type="text/javascript">
```

Comment:
Since we want to use modern JavaScript as much as possible this should be a [module](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules).
```suggestion
    <script type="module">
```

Reply:
Done